attribute_set_completeness/models/attribute_set_owner_mixin.py

```diff
@@ -14,33 +14,50 @@ class AttributeSetOwnerMixin(models.AbstractModel):
         default="not_complete",
         readonly=True,
     )
+
+    is_completion_state_visible = fields.Boolean(compute="_compute_completion_rate")
+    is_completion_rate_visible = fields.Boolean(compute="_compute_completion_rate")
+
```

Comment:
@lmignon I followed your advice and created these 2 booleans filled by the same `_compute_completion_rate()` method.
It's cool because I could had this logic in the tests too. Let me know it's ok for you !